src/test/ui/higher-rank-trait-bounds/issue-62203-hrtb-ice.stderr

```diff
@@ -44,11 +44,12 @@ LL | |             },
 LL | |         },
    | |_________^ expected struct `Unit3`, found struct `Unit4`
    |
-note: required for `L<[closure@$DIR/issue-62203-hrtb-ice.rs:42:16: 42:19]>` to implement `for<'r> T0<'r, (&'r u8,)>`
+note: required for `L<[closure@issue-62203-hrtb-ice.rs:42:16]>` to implement `for<'r> T0<'r, (&'r u8,)>`
   --> $DIR/issue-62203-hrtb-ice.rs:17:16
    |
 LL | impl<'a, A, T> T0<'a, A> for L<T>
    |                ^^^^^^^^^     ^^^^
+   = note: the full type name has been written to '$TEST_BUILD_DIR/higher-rank-trait-bounds/issue-62203-hrtb-ice/issue-62203-hrtb-ice.long-type-6413478884337013991.txt'
```

Comment:
`./x.py test --bless` changed this as shown (the number in the filename changes based on the build hash or something). I don't know how to make it output how it was before, with `$DIR` and without the "full type name" file.